api/build.gradle

```diff
@@ -101,13 +101,14 @@ repositories {   // repositories for Jar's you access in your code
 }
 
 dependencies {
-  providedCompile group: 'javax.servlet', name: 'servlet-api', version:'2.5'
+  providedCompile group: 'javax.servlet', name: 'javax.servlet-api', version:'3.1.0'
+
   compile 'com.google.appengine:appengine:+'
   compile('org.springframework.boot:spring-boot-starter-web:+') {
     exclude module: 'spring-boot-starter-tomcat'
   }
   compile 'org.springframework.security:spring-security-web:+'
-  compile 'io.swagger:swagger-codegen:+'
+
```

Comment:
Waseem this dependency line breaking something? (I added it because I couldn't get VS Code to treat the generated source as valid without it. I can work around this locally if needed.)